poetry.lock

```diff
@@ -1801,6 +1850,35 @@ files = [
     {file = "tomli-2.0.1.tar.gz", hash = "sha256:de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f"},
 ]
 
+[[package]]
+name = "torch"
+version = "2.1.0"
+description = "Tensors and Dynamic neural networks in Python with strong GPU acceleration"
```

Comment:
Can we remove pytorch from the poetry file? We already have it installed with the `poe torch-cuda` command.

Reply:
Also, this is going to install the cpu version.

Reply:
It may be a good idea ! But anyway if you install it and run `poe torch-cuda` then it will be fine. So as you want.

Reply:
Torch is a dependency of the packages manga-ocr that was part of #14. When it is removed from the lock, an error is thrown and so I will not remove it, for our future install to be successful.

![image](https://github.com/cia-ulaval/MangaAutoTranslator/assets/111534376/c4bf89d1-0a92-49a6-a640-71df7fc6c009)

Reply:
That's because you also need to remove the line 747 `torch = ">=1.0"`. I don't know why this is in main, I thought I removed it.

Reply:
So if you remove the explicit dependency of torch + pytorch, it's not gonna flag it as an error (at least it didn't when I did it on my side).

Reply:
I confirm that I did not remove it, and that I should have.